qualification/report/katdoc.sty

```diff
@@ -1,64 +1,84 @@
-% File: katdoc.sty
-% Subject: Stylesheet for KAT document
-% Author: M. Welz and A. Pinska; based on template by R.T. Lord and J.C. Guzman
 \ProvidesPackage{katdoc}
-% \setcounter{errorcontextlines}{999}
-
-\newcommand{\stylesheetVersion}{1.1.2-dirty}
+\newcommand{\stylesheetVersion}{1.1.3-DSP}
 
+\usepackage[utf8]{inputenc}
 \usepackage{graphicx}
 \usepackage{lastpage}
 \usepackage{mathptmx}
 \usepackage[T1]{fontenc}
 \usepackage{geometry}
 \usepackage{array}
 \usepackage{babel}
+\usepackage{csquotes}  % Latex recommends this when babel is used
 \usepackage{fancyhdr}
 \usepackage{tabularx}
 \usepackage{longtable}
 \usepackage{multirow}
 \usepackage{ifthen}
 \usepackage{float}
 \usepackage{enumitem}
+\usepackage{mathtools}
+\usepackage{tablefootnote}
+\usepackage{color}
+\usepackage{colortbl}
 
 % For plotting to work nicely.
-\usepackage[utf8]{inputenc}
+\usepackage{pgf}
```

Comment:
Is this necessary, or will pgfplots suck it in anyway?

Reply:
It was somewhere previously, I just rearranged it.
Turns out it isn't, so I'll remove it.